sound/soc/sof/trace.c

```diff
@@ -539,6 +539,10 @@ EXPORT_SYMBOL(snd_sof_trace_notify_for_error);
 
 void snd_sof_release_trace(struct snd_sof_dev *sdev)
 {
+	struct sof_ipc_fw_ready *ready = &sdev->fw_ready;
+	struct sof_ipc_fw_version *v = &ready->version;
+	struct sof_ipc_cmd_hdr hdr;
+	struct sof_ipc_reply ipc_reply;
```

Comment:
nitpick: can you swap these?

or move them local under the `if(ABI)` ?

Reply:
@ranj063 let's do this change suggested by @ujfalusi  with a fixup